packages/core/src/browser/style/status-bar.css

```diff
@@ -51,4 +51,24 @@
 
 #theia-statusBar .area.right .element {
     margin-left: 15px;
-}
\ No newline at end of file
+}
+
+#theia-statusBar .area.left .element.warning,
+#theia-statusBar .area.right .element.warning {
```

Comment:
You could simplify this selector to
```
#theia-statusBar .element.warning
```
so you don't need to duplicate for left and right. Same for the other styles.

Reply:
Done, thanks!